public/app/core/components/scroll/scroll.ts

```diff
@@ -1,12 +1,17 @@
 import PerfectScrollbar from 'perfect-scrollbar';
 import coreModule from 'app/core/core_module';
+import appEvents from 'app/core/app_events';
 
 export function geminiScrollbar() {
   return {
     restrict: 'A',
     link: function(scope, elem, attrs) {
       let scrollbar = new PerfectScrollbar(elem[0]);
 
+      appEvents.on('scroll-top', () => {
```

Comment:
need to unregister this event on scope destroy, as this is a common requirement you can pass scope as last param and have the emitter register the destroy event and unbind the listener.
https://github.com/grafana/grafana/blob/master/public/app/core/utils/emitter.ts#L19